src/Microsoft.OpenApi.Hidi/OpenApiService.cs

```diff
@@ -198,5 +201,17 @@ internal static void ValidateOpenApiDocument(string input)
 
             Console.WriteLine(statsVisitor.GetStatisticsReport());
         }
+
+        private static OpenApiFormat GetOpenApiFormat(string input)
+        {
+            if (!input.StartsWith("http") && Path.GetExtension(input) == ".json")
```

Comment:
```suggestion
            return !input.StartsWith("http") && Path.GetExtension(input) == ".json" ? OpenApiFormat.Json : OpenApiFormat.Yaml;
```
You can replace the entire `if` with a ternary operator statement.

Reply:
Good suggestion. Thank you.